package.json

```diff
@@ -25,14 +25,16 @@
     "workbox-build": "^4.3.1"
   },
   "devDependencies": {
+    "browserify": "^16.5.0",
```

Comment:
Would be nice to run the browserify not as a global install. Would merge a PR if you can fix that :)